src/Shared/E2ETesting/SeleniumStandaloneServer.cs

```diff
@@ -86,10 +86,20 @@ private static async Task InitializeInstance(ITestOutputHelper output)
             var port = FindAvailablePort();
             var uri = new UriBuilder("http", "localhost", port, "/wd/hub").Uri;
 
+            var seleniumConfigPath = typeof(SeleniumStandaloneServer).Assembly
+                .GetCustomAttributes<AssemblyMetadataAttribute>()
+                .FirstOrDefault(k => k.Key == "Microsoft.AspNetCore.Testing.SeleniumConfigPath")
+                ?.Value;
+
+            if (seleniumConfigPath == null)
+            {
+                throw new InvalidOperationException("Selenium config path not configured. Does this project import the E2ETesting.targets");
+            }
+
             var psi = new ProcessStartInfo
             {
                 FileName = "npm",
-                Arguments = $"run selenium-standalone start -- -- -port {port}",
+                Arguments = $"run selenium-standalone start -- --config \"{seleniumConfigPath}\" -- -port {port}",
```

Comment:
If we're hardcoding the `seleniumConfigPath` into `package.json` anyway, why go to all this trouble to pass it dynamically here? Would it be simpler to modify the `selenium-standalone` script in `package.json` so that it also hardcodes the config path? Then we wouldn't need to pass it from here, and therefore wouldn't need the `AssemblyMetadataAttribute`, and wouldn't need the change to `E2ETesting.props`, etc.

Reply:
I think the issue is that knowing the relative path inside of a test is a bit tricky. Using relative paths can mess up depending on how the test is launched and it's not always correct to assume PWD is the test directory. In other places where we need to find something relative, we do something quite similar - have the build tell us where things are.

Reply:
OK, sounds like a good reason!